src/main/resources/avro/referencemethods.avdl

```diff
@@ -113,9 +113,9 @@ as JSON.
 */
 record SearchReferencesRequest {
   /**
-  If present, return only references which belong to this reference set.
+  The ReferenceSet to search.
   */
-  union { null, string } referenceSetId = null;
```

Comment:
Seems this is breaking a bit from the many to many relationship which has been discussed previously around references : referenceSets. Have the graph changes altered that discussion?

I'm thinking of the CRAM use-case, for example. Given an MD5, find the relevant reference.